weaviate/collection/collection.py

```diff
@@ -374,6 +378,16 @@ def _json_to_object(self, obj: Dict[str, Any]) -> _Object:
             metadata=_MetadataReturn(obj),
         )
 
+    def add_property(self, additional_property: Property):
+        path = f"/schema/{self.name}/properties"
+        obj = additional_property.to_dict()
+        try:
+            response = self._connection.post(path=path, weaviate_object=obj)
+        except RequestsConnectionError as conn_err:
+            raise RequestsConnectionError("Property was created properly.") from conn_err
```

Comment:
Should this read `"Property was not created properly.`?

Reply:
yep - copy and paste from old code